docs/data/material/customization/containerQueries/containerQueries.md

```diff
@@ -0,0 +1,91 @@
+# Container Queries
+
+<p class="description">API that enables the use of CSS container queries based on theme breakpoints.</p>
+
+A utility function based on the [theme breakpoints](/material-ui/customization/breakpoints/) for creating [CSS container queries](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_containment/Container_queries).
```

Comment:
I'd put this sentence as the page's description and then use the first sentence of the "Usage" section to add the MDN link (given the link to theme.breakpoints is already there. And then, remove this line here.

Reply:
Got it.